src/openms/source/FORMAT/IdXMLFile.cpp

```diff
@@ -647,12 +653,12 @@ namespace OpenMS
 
         for (std::vector<String>::const_iterator it = accessions.begin(); it != accessions.end(); ++it)
         {
-          std::map<String, String>::const_iterator it2 = proteinid_to_accession_.find(*it);
+          const auto it2 = proteinid_to_accession_.find(*it);
           if (it2 != proteinid_to_accession_.end())
           {
             PeptideEvidence pe;
             pe.setProteinAccession(it2->second);
-            peptide_evidences_.push_back(pe);
+            peptide_evidences_.push_back(std::move(pe));
```

Comment:
is PeptideEvidence moveable?

Reply:
yes it is. If it were not, this would just make it an r-value and copy assignment would kick in.